com.civicrm.hrjobroles/CRM/Hrjobroles/Import/Form/DataSource.php

```diff
@@ -60,8 +58,12 @@ public function preProcess() {
   public function buildQuickForm() {
     //Setting Upload File Size
     $config = CRM_Core_Config::singleton();
-
-    $uploadFileSize = CRM_Core_Config_Defaults::formatUnitSize($config->maxFileSize.'m');
+    if ($config->maxImportFileSize >= 8388608) {
+      $uploadFileSize = 8388608;
+    }
+    else {
+      $uploadFileSize = $config->maxImportFileSize;
+    }
```

Comment:
Well... if the default size in PHP, it should be set in php.ini not here. One more reason to get rid of this